force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -308,7 +419,7 @@ public with sharing class FlowControlLoopsCollections {
         // add Pears => 20
         // add Grapes => 25
 
-        return null; // Replace null with the variable you used to store the result
+        return addItemsToMap ; // Replace null with the variable you used to store the result
```

Comment:
Good work. It you want to use a more concise way can be to just use the build in Map constructor. 

```
  public static Map<String, Integer> addItemsToMap() {

        return new Map<String, Integer>{
                'Apples' => 5,
                'Oranges' => 10,
                'Bananas' => 15,
                'Pears' => 20,
                'Grapes' => 25
        };
    }
 ```